src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -84,6 +84,11 @@ internal Version GetAssemblyVersion()
             return typeof(CommonCompiler).GetTypeInfo().Assembly.GetName().Version;
         }
 
+        internal string GetCultureName()
```

Comment:
Probably would have been nicer, but not going to bother resetting CI over that. The function just above is a 1-liner too so I'll argue file self-consistency + minimal diff. ;)